plugins/discovery-gce/build.gradle

```diff
@@ -23,7 +23,6 @@ dependencies {
   api "com.google.http-client:google-http-client-gson:${versions.google}"
   api 'com.google.code.findbugs:jsr305:3.0.2'
   api "commons-logging:commons-logging:${versions.commonslogging}"
-  api "org.apache.logging.log4j:log4j-1.2-api:${versions.log4j}"
```

Comment:
I wonder if this is safe? we probably don't use log4j-1.2 in this plugin, so this is required by some of its dependencies. We don't have bridges from log4j to es logging yet

Reply:
Hmm, not any more it seems. I've reverted this.